next.config.js

```diff
@@ -146,7 +109,8 @@ module.exports = withSwingset({
       ENABLE_VERSIONED_DOCS: process.env.ENABLE_VERSIONED_DOCS || false,
       HASHI_ENV: process.env.HASHI_ENV || 'development',
       IS_CONTENT_PREVIEW: process.env.IS_CONTENT_PREVIEW,
-      SEGMENT_WRITE_KEY: process.env.SEGMENT_WRITE_KEY,
+      // TODO: was in `assembly-ui-v1`, needed in `main`?
+      // SEGMENT_WRITE_KEY: process.env.SEGMENT_WRITE_KEY,
```

Comment:
@BRKalow @dstaley @zchsh do y'all know if `SEGMENT_WRITE_KEY` is needed in `main`?

Reply:
@ashleemboyer They live in the product data files for each specific product for the io sites, but this one might be necessary for the actual DevDot app.

Reply:
The naming style is different so I don't see potential for clashing, I'll uncomment!

Reply:
Updated in https://github.com/hashicorp/dev-portal/pull/175/commits/60356478f12bac9d0481367cb20babbaa988d47e